app/assets/javascripts/discourse/app/components/form-template-field/input.hbs

```diff
@@ -16,7 +15,7 @@
     placeholder={{@attributes.placeholder}}
     required={{if @validations.required "required" ""}}
     pattern={{@validations.pattern}}
-    minLength={{@validations.min}}
-    maxLength={{@validations.max}}
+    minlength={{@validations.minimum}}
+    maxlength={{@validations.maximum}}
   />
```

Comment:
Changing from `min`/`max` to `minimum`/`maximum` so its consistent with what is shown as the desired key for users in the validations modal.

<img width="705" alt="Screenshot 2023-08-04 at 11 02 21" src="https://github.com/discourse/discourse/assets/30090424/6f0be61d-a32b-4c72-abfe-4d40508ef019">